README.md

```diff
@@ -338,10 +338,43 @@ You can check "ifconfig -a" to check list of interfaces. Ethernet, WiFi and 6loW
 
         $root@OpenWrt:/# /etc/init.d/network restart
 
+##Writing UBIFS image to Flash
```

Comment:
Move this section above the line "You should see the logs on the console as below:" i.e. below "Configure Uboot" sub-section

Also, please add a section "Configure Network" above the "You should see the logs on the console as below:"